src/test/java/org/apache/pulsar/io/jcloud/format/JsonFormatTest.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This change causes this warning and error: https://github.com/streamnative/pulsar-io-cloud-storage/actions/runs/4552976797/jobs/8028982534?pr=626#step:4:25543